packages/firestore/src/remote/stream_bridge.ts

```diff
@@ -63,25 +63,25 @@ export class StreamBridge<I, O> implements Stream<I, O> {
 
   callOnOpen(): void {
     assert(
-      this.wrappedOnOpen !== null,
-      'Cannot call onOpen because no callback ' + 'was set'
+      this.wrappedOnOpen !== undefined,
+      'Cannot call onOpen because no callback was set'
     );
-    this.wrappedOnOpen();
+    this.wrappedOnOpen!();
```

Comment:
FWIW, I always feel bad about myself when I use `!` and so I write something like:
```
if (this.wrappedOnOpen === undefined) {
  fail('Cannot call onOpen because no callback was set');
} else {
  this.wrappedOnOpen();
}
```

You could also do `return fail(...)` and drop the else block.

If you don't feel bad about yourself though, that's cool.

Reply:
This is very confined and follows the existing code, so I am inclined to leave it as is.